src/resources/js/move-tickets-dialog.js

```diff
@@ -248,17 +248,32 @@ var tribe_move_tickets = tribe_move_tickets || {};
 
 				// Clear the existing list
 				$post_choices.html( '' );
-				let total_posts = 0;
-
-				for ( const key in response.data.posts ) {
-					const post_id = parseInt( key, 10 );
-					const title = response.data.posts[ key ];
-					total_posts++;
-
-					$post_choices.append(
-						'<label> <input type="radio" value="' + post_id + '" name="post-choice">' + title + '</label>'
-					);
-				}
+				var total_posts = 0;
+                
+                // Convert the object to a sortable array
+                var posts_array = [];
+                for (var key in response.data.posts) {
+                    posts_array.push({
+                        id: key,
+                        title: response.data.posts[key]
+                    });
+                }
+                
+                // Sort by title alphabetically
+                posts_array.sort(function(a, b) {
+                    return a.title.localeCompare(b.title);
+                });
+
+                // Use the sorted array to build the options
+                for (var i = 0; i < posts_array.length; i++) {
+                    var post_id = parseInt(posts_array[i].id, 10);
+                    var title = posts_array[i].title;
+                    total_posts++;
+
+                    $post_choices.append(
+                        '<label> <input type="radio" value="' + post_id + '" name="post-choice">' + title + '</label>'
```

Comment:
You can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here instead of the `+` operator.

```suggestion
                        `<label><input type="radio" value="${post_id}" name="post-choice" />${title}</label>`
```